### DISCUSSION TITLE
Accessing more options without pressing or clicking

### DISCUSSION THREAD

**Discussion by hasantahir** [godbout/Wooshy.docs#111] [General]
For example, Wooshy identifies a keyboard shortcut of `⌃ + ⇧ ⌥ + M` in MS Word for further options. Is there a way to access those options without clicking first? What I see is the Wooshy bar being resized.

Cheers

https://user-images.githubusercontent.com/11236148/215109633-dd55ccb7-bc52-4667-9738-783d1ffec23a.mov

**Comment by godbout** [comment]
i've deleted Word yesterday 😂️ ( 😭️ )

**Comment by godbout** [comment]
first thing i see is that `control option m` under macOS creates a new linefeed. not just in Wooshy. this is standard and you can see this behavior in Notes, for example.

i'll check if Word overrides that or it's just bullshitting.

**Reply by godbout** [reply]
i can't try now because i've already got the free month subscription before and i'm not ready to give money to Microsoft yet. can you check what `control option shift m` does when you're not in Wooshy, and share a screencast? thanks.

**Reply by godbout** [reply]
but something is gonna be sure, when you're in the Wooshy Text Input, the hotkeys you gonna press are gonna be received by Wooshy, not any app behind. if Wooshy doesn't recognize the hotkey, it will let it passthrough, and macOS will get it and handle it accordingly. in the case you're showing in your screencast, well, this is what happens. Wooshy lets the hotkey through, and for macOS `control option m` means `create a linefeed` (with or without the shift). so that's what it does. macOS will not send the hotkey to Word, which you may have expected.

**Comment by hasantahir** [comment]
Thanks, yeah the shortcut without Wooshy moves the cursor to a new line. I thought one would be able to access _more options_ using that key 😮. Thanks 

https://user-images.githubusercontent.com/11236148/215260981-65c67121-9f81-402c-aa91-03d378ee2eae.mov

**Reply by godbout** [reply]
wait. so basically Microsoft tells you to use a shortcut to reach those options, but actually the shortcut doesn't work? 😂️😂️😂️
